apps/devportal/data/markdown/pages/learn/faq/xm-personalize/customization.md

```diff
@@ -6,14 +6,14 @@ hasSubPageNav: true
 cdpTags: ['xm', 'personalize']
 ---
 
-**Can I extend XP2XMP websites using my existing .NET MVC component library?**
+### Can I extend XP2XMP websites using my existing .NET MVC component library?
 
 MVC websites that are self-hosted or run in Sitecore Managed Cloud with the XP2XMP product bundle should continue to support existing .NET MVC components.
 
-**We use Sitecore Search and have customized the indexes. Will this work in XP2XMP and XM Cloud?**
+### We use Sitecore XM's content search API and have customized the indexes. Will this work in XP2XMP and XM Cloud?
 
 Customers will be able to create custom indexes if they host Sitecore Experience Manager in their own Cloud or use Sitecore Managed Cloud. With XM Cloud, customers will not be able to create custom indexes as Solr will be fully controlled by the XM Cloud platform. Sitecore recommends that customers use a third-party search provider if custom indexes are required.
 
-**What extension possibilities/flexibility for custom business requirements are there for XM, Send, Personalize?**
+### What extension possibilities/flexibility for custom business requirements are there for XM, Send, Personalize?

```

Comment:
Another mention to Send, not sure it makes sense in an XM + Personalize document, since it has it's own considerations.